application.py

```diff
@@ -613,10 +617,14 @@ def create_graph(output_cellset, includeisolated=False, graphname="cellgraph"):
     """
     global e
```

Comment:
unrelated to this diff: what's this global?

Reply:
Part of how EM2 works... Working on refactoring it